cli/js/web/headers.ts

```diff
@@ -63,23 +72,13 @@ class HeadersBase {
             2
           );
 
-          const [name, value] = normalizeParams(tuple[0], tuple[1]);
-          validateName(name);
-          validateValue(value);
-          const existingValue = this[headerMap].get(name);
-          this[headerMap].set(
-            name,
-            existingValue ? `${existingValue}, ${value}` : value
-          );
+          this.append(tuple[0], tuple[1]);
```

Comment:
I refactored the constructor a bit, to keep the code `DRY` and to reuse `Set-Cookie` logic in `append/set` methods.